package/views.py

```diff
@@ -28,6 +28,16 @@
 from package.models import Category, FlaggedPackage, Package, PackageExample
 from package.repos import get_all_repos
 from package.tables import PackageTable
+from searchv2.rules import calc_package_weight
+from searchv2.rules import DeprecatedRule
+from searchv2.rules import DescriptionRule
+from searchv2.rules import DownloadsRule
+from searchv2.rules import ForkRule
+from searchv2.rules import LastUpdatedRule
+from searchv2.rules import RecentReleaseRule
+from searchv2.rules import ScoreRuleGroup
+from searchv2.rules import UsageCountRule
+from searchv2.rules import WatchersRule
```

Comment:
I think it makes code change diffs more readable. I have had several clients in a row prefer it and I think I'm just used to the config. I don't have a store feeling either way but I I think it might be a default in the config. (I might need to double check this.)